src/targets/adminCreateUser.ts

```diff
@@ -105,7 +105,13 @@ export const AdminCreateUser =
   async (ctx, req) => {
     const userPool = await cognito.getUserPool(ctx, req.UserPoolId);
     const existingUser = await userPool.getUserByUsername(ctx, req.Username);
-    if (existingUser && req.MessageAction === "RESEND") {
+    const supressWelcomeMessage =
+      req.MessageAction === "SUPRESS";
+
+    const resendWelcomeMessage = 
```

Comment:
Do we need the resend message constant here? We only dealing with the suppress message. Maybe remove this and add it when we are ready for resend message functionality.

Reply:
`if (existingUser && resendWelcomeMessage) {`

i just created a var for the check in the line 108

if (existingUser && req.MessageAction === "RESEND") {

so it reads better